### PR TITLE
Fixing documentation

### PR DESCRIPTION
**Pull request by lnfnunes**
Just fixing readme default path reference

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ To install via Package Control, do the following:
 
 - JSCSFormatter: Format this file
 
+**Shortcut key:**
+
+- Ctrl+Shift+H
```
Comment (elicwhite) [inline README.md]: This is `Ctrl + Shift + H` on Linux and Windows, but `Cmd + Shift + H` on Mac.

**Comment by elicwhite** [comment]
Thanks for contributing! Looks good other than my one comment.

**Comment by lnfnunes** [comment]
Changed as mentioned... tks

**Comment by elicwhite** [comment]
Thank you for helping improve the docs.

**Pull request merged**